src/configuration/config.py

```diff
@@ -0,0 +1,241 @@
+"""Module for configuration logic"""
+from abc import ABC, ABCMeta, abstractproperty, abstractmethod, abstractclassmethod, abstractstaticmethod
+
+import configparser
+import yaml
+import json
+import os
+import local_settings
+from src.helpers.validator import Validator
+
+
+class Config:
```

Comment:
The general purpose of this class should be to read the configuration from different sources like YAML file or environment variable. The common cases are:
- You run your tests from Jenkins job with some choices like browser (Chrome, Firefox etc.) It is added to the environment variables. From your test framework you read this configuration
- You keep some environment configuration in YAML like url, user accounts and you need to read the correct configuration depending on environment choice (qa, staging, prod)
It's a rare case when you need to set the env variable or write the config to YAML.